src/foam/nanos/blob/FileService.java

```diff
@@ -54,6 +57,15 @@ protected void download(X x) {
         return;
       }
 
+      //Replace @version@ with actual foam version
+      if (file.getMimeType().equals("text/html")) {
```

Comment:
if ( "text/html".equals(file.getMimeType()) ) {
is slightly better because it avoids possibility of NPE (not likely in this case, but in general, use the literal as 'this' to avoid the possibility. Also, fixes the spacing with spaces after ( and before ).